src/classes/HiFiMixerSession.ts

```diff
@@ -1069,18 +1075,9 @@ export class HiFiMixerSession {
     }
 
     /**
-     * @param currentHifiAudioAPIData - The new user data that we want to send to the High Fidelity Audio API server.
-     * @returns If this operation is successful, returns `{ success: true, stringifiedDataForMixer: <the raw data that was transmitted to the server>}`. If unsuccessful, returns
-     * `{ success: false, error: <an error message> }`.
+     * This method converts the HiFiAudioAPIData structure into the format needed by the mixer.
      */
-    _transmitHiFiAudioAPIDataToServer(currentHifiAudioAPIData: HiFiAudioAPIData, previousHifiAudioAPIData?: HiFiAudioAPIData): any {
-        if (!this.mixerInfo["connected"] || !this._raviSession) {
-            return {
-                success: false,
-                error: `Can't transmit data to mixer; not connected to mixer.`
-            };
-        }
-
+    _getDataToTransmitToMixer(currentHifiAudioAPIData: HiFiAudioAPIData, previousHifiAudioAPIData?: HiFiAudioAPIData): any {
```

Comment:
Place all of the code that calculates the JSON to send to the mixer in someplace multiple pieces of code can get to.